dvc/command/version.py

```diff
@@ -38,24 +42,32 @@ def run(self):
             binary=binary,
         )
 
-        try:
-            repo = Repo()
-            root_directory = repo.root_dir
+        if psutil:
+            try:
+                repo = Repo()
+                root_directory = repo.root_dir
+
+                info += (
+                    "Cache: {cache}\n"
+                    "Filesystem type (cache directory): {fs_cache}\n"
+                ).format(
+                    cache=self.get_linktype_support_info(repo),
+                    fs_cache=self.get_fs_type(repo.cache.local.cache_dir),
+                )
 
-            info += (
-                "Cache: {cache}\n"
-                "Filesystem type (cache directory): {fs_cache}\n"
-            ).format(
-                cache=self.get_linktype_support_info(repo),
-                fs_cache=self.get_fs_type(repo.cache.local.cache_dir),
-            )
+            except NotDvcRepoError:
+                root_directory = os.getcwd()
 
-        except NotDvcRepoError:
-            root_directory = os.getcwd()
+            info += ("Filesystem type (workspace): {fs_root}").format(
+                fs_root=self.get_fs_type(os.path.abspath(root_directory))
+            )
+        else:
+            logger.warning(
```

Comment:
let's do debug, let's not show these warning to end users

Reply:
Well, otherwise we might get bug reports, since https://dvc.org/doc/commands-reference/version is saying that it should show that info. I would much rather leave the warning, especially since `dvc version` is not a usual workflow command.

Reply:
it's terrible from UI/UX and not useful to show it. We can change docs for now - one line that mentions that you have to install psutil to get those results.

Reply:
@shcheklein Then I don't really see any reason for that message to exist at all. It is useless in debug mode, since users don't typically use it and for developers it is pretty clear already.

Reply:
Submitted https://github.com/iterative/dvc.org/pull/502